ezyfox-server-core/src/main/java/com/tvd12/ezyfoxserver/socket/EzySimplePacket.java

```diff
@@ -25,10 +30,14 @@ public void setFragment(Object fragment) {
 
     @Override
     public int getSize() {
-        if (data instanceof String) {
-            return ((String) data).length();
+        Object currentData = data;
+        if (currentData == null) {
+            return 0;
+        }
+        if (currentData instanceof String) {
+            return ((String) currentData).length();
         }
-        return ((byte[]) data).length;
+        return ((byte[]) currentData).length;
     }
```

Comment:
Why do we need a new variable `currentData` here? I guess we can use `data` directly

Reply:
We have:

```
public void release() {
   this.data = null;
   this.released = true;
}
```

So the data can be null, we need to copy reference of data to avoid null pointer exception

Reply:
Yep, sounds good to me